EditingStyle: rename forceInline to forceDisplayInline

vitorroriz opened the pull request:
#### 4712df393b97e76b3a98c2a98d634e59ed087f7b
<pre>
EditingStyle: rename forceInline to forceDisplayInline
<a href="https://bugs.webkit.org/show_bug.cgi?id=297003">https://bugs.webkit.org/show_bug.cgi?id=297003</a>
<a href="https://rdar.apple.com/157670437">rdar://157670437</a>

Reviewed by Sammy Gill.

&quot;forceInline&quot; is not a great name, it is misleading because in the
EditingStyle context &quot;inline&quot; is often used to describe inline style
of elements. It also doesn&apos;t express the fact that it is related to the
CSS display property.

* Source/WebCore/editing/EditingStyle.cpp:
(WebCore::EditingStyle::forceDisplayInline):
(WebCore::EditingStyle::forceInline): Deleted.
* Source/WebCore/editing/EditingStyle.h:
* Source/WebCore/editing/markup.cpp:
(WebCore::StyledMarkupAccumulator::appendText):
(WebCore::StyledMarkupAccumulator::appendStartTag):

Canonical link: <a href="https://commits.webkit.org/298296@main">https://commits.webkit.org/298296@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/b6ab35959d8de24e6b788bf7456520ee063a15e6

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/115039 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/34755 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/25236 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/121159 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/65688 "Built successfully") | [✅ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/9a0fb378-92dc-4a1a-9d3b-4e8819e4c65e) 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/116928 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/35400 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/43316 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/87419 "Passed tests") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/60/builds/42228 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/479c19c6-1576-42d8-af45-2506856be0b9) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/117987 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/28216 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/103281 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/67815 "Passed tests") | | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/27390 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/21401 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/64811 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/97603 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/21518 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/124349 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/42012 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/31413 "Passed tests") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/96221 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/42384 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/99471 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/96006 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/24431 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/41206 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/19043 "Passed tests") | [  ~~🛠 playstation~~](https://ews-build.webkit.org/#/builders/134/builds/38020 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/41887 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/47422 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/41431 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/44748 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/43167 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

sammygill reviewed (approved):
`forceDisplayInline` or `forceInlineDisplay`?

vitorroriz committed:
EditingStyle: rename forceInline to forceDisplayInline
https://bugs.webkit.org/show_bug.cgi?id=297003
rdar://157670437

Reviewed by Sammy Gill.

"forceInline" is not a great name, it is misleading because in the
EditingStyle context "inline" is often used to describe inline style
of elements. It also doesn't express the fact that it is related to the
CSS display property.

* Source/WebCore/editing/EditingStyle.cpp:
(WebCore::EditingStyle::forceDisplayInline):
(WebCore::EditingStyle::forceInline): Deleted.
* Source/WebCore/editing/EditingStyle.h:
* Source/WebCore/editing/markup.cpp:
(WebCore::StyledMarkupAccumulator::appendText):
(WebCore::StyledMarkupAccumulator::appendStartTag):

Canonical link: https://commits.webkit.org/298296@main
